modular/gater/request_context.go

```diff
@@ -40,6 +41,19 @@ type RequestContext struct {
 	startTime time.Time
 }
 
+var skipAuthRouterNames = []string{
+	requestNonceRouterName,
+	updateUserPublicKeyRouterName, // this will skip general auth algorithms first and use specific "personal sign" later
+	downloadObjectByUniversalEndpointName,
```

Comment:
why `downloadObject` can skip auth?

Reply:
universal endpoints don't need authentication(V1 or Off-chain-Auth).  For private objects, they use a personal-sign style authentication.